phylotree_loader_cmd.drush.inc

```diff
@@ -94,6 +126,29 @@ function drush_phylotree_loader_cmd_phylotree($dir, $analysis_id)
 
             drupal_set_message($error, 'error');
         }
+
+        $query = chado_db_select('phylotree', 'pt');
+        $query->distinct();
+        $query->fields('cv', ['name', 'definition']);
+        $query->fields('c', ['name']);
+        $query->join('phylonode', 'pn', 'pn.phylotree_id = pt.phylotree_id');
+        $query->join('feature_cvterm', 'fc', 'fc.feature_id = pn.feature_id');
+        $query->join('cvterm', 'cv', 'fc.cvterm_id = cv.cvterm_id');
+        $query->join('cv', 'c', 'c.cv_id = cv.cv_id');
+        $query->condition('pt.name', $unique_name, '=');
+
+        $results = $query->execute()->fetchAll();
+
+        $match = ['name' => $unique_name];
+        $values = ['comment' => phylotree_make_cvterm_table($results)];
```

Comment:
We also want the download table using `phylotree_make_download_table()`. Also make `phylotree_make_cvterm_table` return an empty string if results is empty so we don't end up with an empty table.